src/FunderMaps.Data/Repositories/IncidentRepository.cs

```diff
@@ -255,6 +262,8 @@ public override async ValueTask UpdateAsync(Incident entity)
                 throw new ArgumentNullException(nameof(entity));
             }
 
+            ResetCacheEntity(entity);
```

Comment:
Is it worth it to get the updated item and cache it right away?

Reply:
Nope, wrong responsibility